internal/fs/inode/dir.go

```diff
@@ -297,7 +298,8 @@ func findExplicitInode(ctx context.Context, bucket *gcsx.SyncerBucket, name Name
 		Name: name.GcsObjectName(),
 	}
 
-	o, err := bucket.StatObject(ctx, req)
+	m, _, err := bucket.StatObject(ctx, req)
+	o := storageutil.ConvertMinObjectToObject(m)
```

Comment:
can we also make changes in core.go to use MinObject inside Core struct also and use min object here as well ?

Reply:
This will require a lot of changes. I'll check and raise a separate PR for this change.